localisation/wc_religions.csv

```diff
@@ -134,8 +134,6 @@ elemental_lords_group;Elementalist;Elementalist;Elementalist;;;;;;;;;;;x
 orcish_shamanism;Ancestral;ancestrale;Ahnenverehrung;;;;;;;;;;;x
 orcish_shamanism_DESC;Ancestral worship is one of the faiths in Azeroth and one of the main tenets of shamanism practices in the world. Most races respect the dead, but those who worship their ancestors, like orcs and tauren, believe that the fallen possess powers, which the living can use. Tauren speak to them and can request their help through dreaming and such. Orcs leave more concrete reminders of their ancestors. They carve memorials to the fallen dead and place stones around permanent burial grounds. There are many rituals one can perform, alone or in a group.;;Die Ahnenverehrung ist eine der Glaubensrichtungen in Azeroth und einer der Hauptgrunds�tze der schamanistischen Praktiken in der Welt. Die meisten Rassen respektieren die Toten, aber diejenigen, die ihre Vorfahren verehren, wie Orks und Tauren, glauben, dass die Gefallenen Kr�fte besitzen, welche die Lebenden nutzen k�nnen. Tauren sprechen zu ihnen und k�nnen sie durch Tr�ume und dergleichen um Hilfe bitten. Orks hinterlassen handfestere Erinnerungen an ihre Vorfahren. Sie mei�eln Denkm�ler f�r die gefallenen Toten und errichten Steine um dauerhafte Grabst�tten herum. Es gibt viele Rituale, die allein oder in der Gruppe durchgef�hrt werden k�nnen.;;;;;;;;;;;x
 EVIL_SPIRITS;the Evil Spirits;les Esprits Mal;die b�sen Geister;;;;;;;;;;;x
-primitive_shamanism;Fetishistic;F�tichiste;Fetischistisch;;;;;;;;;;;x
-primitive_shamanism_DESC;The worshipping of primitive godlike figures, as the bear twins Ursoc and Ursol, is uncommon in Azeroth, however, their followers primarily revere nature and the spirits of the world much like "modern" shamanism. The bear twins are the primary patrons of those who believe in a balance between nature and all living creatures. The furbolg are the most fervent followers of the bear twins in the present day, following in their magical tradition and carving their fortress, Timbermaw Hold, in the form of a gigantic, growling bear head.;;Die Verehrung primitiver, gott�hnlicher Gestalten, wie der B�renzwillinge Ursoc und Ursol, ist in Azeroth nicht allzu verbreitet, ihre Anh�nger verehren jedoch in erster Linie die Natur und die Geister der Welt, �hnlich wie der �moderne� Schamanismus. Die B�renzwillinge sind die prim�ren Schirmherren derer, die an ein Gleichgewicht zwischen der Natur und allen Lebewesen glauben. Die Furbolgs sind die gl�hendsten Anh�nger der B�renzwillinge in der heutigen Zeit. Sie folgen ihrer magischen Tradition und schnitzten ihre Festung, die Holzschlundfeste, in Form eines gigantischen, knurrenden B�renkopfes.;;;;;;;;;;;x
```

Comment:
Wait, why they're removed?

Reply:
I went on a limb figuring the variety of elemental/spiritual worship is going to patron gods and the like. As shown with the Quilboars, Kobolds, and Furbolgs. With what I know there isn't anything to place in the description when these are getting their own. Is the intention to use this religion to envelope all those and have it used side by side with them?

Reply:
> I went on a limb figuring the variety of elemental/spiritual worship is going to patron gods and the like. As shown with the Quilboars, Kobolds, and Furbolgs. With what I know there isn't anything to place in the description when these are getting their own. Is the intention to use this religion to envelope all those and have it used side by side with them?

The religion is a placeholder for races that don't have any gods and simply worship primitive spirits or elementals (like gnolls).

Reply:
Modifications made.